bulk_deploy/tsuru_deploy_client.rb

```diff
@@ -0,0 +1,104 @@
+require 'fileutils'
+
+class TsuruDeployClient
+
+  def initialize(
+    api_client:, logger:, environment:, host:, tsuru_home:, protocol: "https://"
+  )
+    target = URI.parse(protocol + environment + "-api." + host)
+
+    targets_path = File.join(ENV["HOME"], ".tsuru_targets")
```

Comment:
If we are using the `tsuru` command line, I think it is better to parse the output of the command `tsuru target-list` rather than reading the status files of the command line tool. 

The files `~/.tsuru_target` and  `~/.tsuru_targets` are used internally by the `tsuru` command line tool, the right interface with the user is the command itself, if we read them directly we are creating a dependency with the command line internal functionality.